app/ssttpaffordability/AffordabilityController.scala

```diff
@@ -212,4 +212,12 @@ class AffordabilityController @Inject() (
       Ok(views.set_up_a_payment_plan_with_an_adviser(isWelsh))
     }
   }
+
+  def getCannotAffordPlan: Action[AnyContent] = as.authorisedSaUser.async { implicit request =>
```

Comment:
this could be a `val` since it doesn't take any parameters

Reply:
Ok